data/map.txt

```diff
@@ -27087,6 +27212,14 @@ planet "Esquiline"
 	security 0
 	government Uninhabited
 
+planet Everhope
+	attributes station hai
+	landscape land/space6
+	description `This station consistently has a large number of Hai warships docked at it, and further investigation reveals that it serves as the principle supply of fuel for the Hai military, as well as the staging point for resupplying ships heading to defend Wah Ki from the Unfettered. A monument beyond the docking area entrance attests to its history performing a similar role during the Korath raids before that.`
+	spaceport `The spaceport is dominated by its function as a military station, though it still has a dual-level promenade like its civilian counterparts of Ochrescoop and Greymoon. Several areas are restricted, but there are also freely accessible pubs and eateries, as well as an impressive cinema and a warehousing section for trading goods. The flooring is textured to prevent any slips and appears to be fairly new, while the rest of the station has a polished but well-worn feel to it.`
```

Comment:
```suggestion
	spaceport `The spaceport is dominated by its function as a military station, though it still has a dual-level promenade like its civilian counterparts of Ochrescoop and Greymoon. Several areas are restricted, but there are also freely accessible pubs and eateries, as well as an impressive cinema and a warehousing section for trading goods. The flooring is textured to prevent slipping and appears to be fairly new, while the rest of the station has a polished but well-worn feel to it.`
```